doc/nrf/ug_matter_configuring.rst

```diff
@@ -29,14 +25,30 @@ The following figure shows the protocol types and the available Matter controlle
 
    Protocol types and controllers used by Matter
 
-To enable the IPv6 communication with the Matter accessory device over the Thread network, the Matter controller requires the Thread Border Router.
-This is because the Matter controller types described on this page do not have an 802.15.4 Thread interface.
-The Border Router bridges the Thread network with the network interface of the controller, for example Wi-Fi.
+Matter development environment depends on the following criteria:
 
-Matter development environment setup options depend on how you run the Matter controller.
-You can either run it on the same device as the Thread Border Router or run the Matter controller and the Thread Border Router on separate devices.
+* Network type:
 
-Depending on your choice, you can set up one of the following development environments:
+  * To enable the IPv6 communication with the Matter accessory device over the Thread network, the Matter controller requires the Thread Border Router.
+    This is because the Matter controller types described on this page do not have an 802.15.4 Thread interface.
+    The Border Router bridges the Thread network with the network interface of the controller, for example Wi-Fi.
+    Watch the following video for an uoverview of the Matter over Thread setup.
+
+    .. raw:: html
+
+       <div style="text-align: center;">
+       <iframe width="560" height="315" src="https://www.youtube.com/embed/XWWJYraNZPs" title="Matter development environment video overview" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+       </div>
+
+
+  * To enable the IPv6 communication with the Matter accessory device over the Wi-Fi network, no additional configuration is required, because the Matter controller can interact with any Wi-Fi device directly and no additional bridging is needed.
```

Comment:
Well the access point is needed to be clear, and Matter controller and a node must be in the same network